src/sphmultipcqueue.h

```diff
@@ -67,11 +67,11 @@ if (handle)
 *
 *	The consumer can access queue entries once they are marked complete.
 *	The consumer:
-*	- checks (spins) for the next allocated entry to become complete.
-*	- uses the returned entry handle to directly access the entry contents.
+*	- checks (spins) for the next allocated entry to become complete
+*	- uses the returned entry handle to directly access the entry contents
 *	- When done processing the queue entry, it marks the entry header
-*	invalid and deallocates the entry.
-*	- This makes the next queue entry available, if any.
+*	invalid and deallocates the entry
+*	- This makes the next queue entry available, if any
```

Comment:
Are these changes really intentional?
Looks like you're removing valid periods '.' here.